src/iop/filmicrgb.c

```diff
@@ -309,7 +309,7 @@ const char *name()
 
 int default_group()
 {
-  return IOP_GROUP_TONE;
+  return IOP_GROUP_TONE | IOP_GROUP_TECHNICAL;
```

Comment:
Could this be the actual problem? This is default_group() and maybe it expect a single group there. Maybe worth double checking?

Reply:
yes, you're right, this comes from here I think : https://github.com/darktable-org/darktable/blob/43533544083436996cfa31066b53593660f146b1/src/tests/unittests/iop/test_filmicrgb.c#L68
Effectively, each iop now belong to 2 groups : the "old" one for the 5 tabs layout and the "new" one for the 3 tabs layout. But in the unit test, we test only for "old" group...

I've no idea how the tests works... Do you think it's enough to just replace `IOP_GROUP_TONE` by `IOP_GROUP_TONE | IOP_GROUP_TECHNICAL` in the test ? or do we need to take in account something more ?

Reply:
I suppose so, yes.

Reply:
done